run-17495340446-artifacts/scripts/audit_style.py

```
@@ -0,0 +1,389 @@
+    elif issue_type == "japanese_residual":
+        return "Re-translate remaining Japanese"
+    elif issue_type == "unbalanced_tags":
+        return f"Balance [{issue.get('tag', '')}] tags"
+    elif issue_type in ["title_too_long", "bullet_too_long"]:
+        return "Condense text or apply expansion policy"
+    
+    return "Review manually"
+
+def should_fail_ci(audit_results: Dict[str, List[Dict[str, Any]]], 
+                   max_issues: int = 10, 
+                   critical_categories: List[str] = None) -> Tuple[bool, str]:
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)